train.py

```diff
@@ -166,7 +176,7 @@ def train(dataset, test_data, n_instances, batch_size, epochs, model, optimizer,
                         help="Embedding dimension. Used for both token IDs and position")
     parser.add_argument('--attention_heads', type=int, default=8)
     parser.add_argument('--max_neighbors', type=int, default=30)
-    parser.add_argument('--epochs', type=int, default=100)
+    parser.add_argument('--epochs', type=int, default=10)
     parser.add_argument('--reports_per_epoch', type=int, default=4)
     parser.add_argument('--lr', type=float, default=0.001)
     parser.add_argument('--test_size', type=int, default=0.1)
```

Comment:
For the defaults here, refer to the dict above like `parser.add_argument('--epochs', type=int, default=hyperparameter_defaults['epochs'])`

Reply:
Cool